packages/admin-panel/src/routes/externalData/externalDatabaseConnections.js

```diff
@@ -54,7 +53,8 @@ const COLUMNS = [
   {
     Header: 'Test',
     type: 'testDatabaseConnection',
-    colWidth: '6.5rem',
```

Comment:
Was this not working previously? Or preferring to switch to static width because it's related to expected character lengths of this field?

Reply:
Since the columns are resizable, we rely now on `react-table` to give use the width, and they only support `px` values, not strings (e.g `rems`) so have changed to a `width` and a pixel value instead